### PR TITLE
CEDS-1958 Align Controller Style with Movements

### PR DESCRIPTION
**Pull request by ed0906**
- Mostly just a rename of `MovementRepository` =>`CacheRepository` to match Movements

- Some Controller refactoring

### CODE REVIEW

File: app/controllers/consolidations/AssociateUCRController.scala [hunk 1]
```diff
@@ -24,30 +24,27 @@ import models.ReturnToStartException
...
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  mcc: MessagesControllerComponents,
-  movementRepository: MovementRepository,
-  associateUcrPage: associate_ucr
+                                        authenticate: AuthenticatedAction,
+                                        getJourney: JourneyRefiner,
+                                        mcc: MessagesControllerComponents,
+                                        movementRepository: CacheRepository,
+                                        associateUcrPage: associate_ucr
```
Comment (PatrykRudnicki) [inline app/controllers/consolidations/AssociateUCRController.scala]: Please run `scalafmt`

File: app/controllers/consolidations/DisassociateUCRSummaryController.scala [hunk 2]
```diff
@@ -32,12 +32,12 @@ import scala.concurrent.ExecutionContext
...
-  getJourney: JourneyRefiner,
-  mcc: MessagesControllerComponents,
-  submissionService: SubmissionService,
-  movementRepository: MovementRepository,
-  page: disassociate_ucr_summary
+                                                  authenticate: AuthenticatedAction,
+                                                  getJourney: JourneyRefiner,
+                                                  mcc: MessagesControllerComponents,
+                                                  submissionService: SubmissionService,
+                                                  movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/consolidations/DisassociateUCRSummaryController.scala]: What about name here? should it be `cache`?

File: app/controllers/consolidations/MucrOptionsController.scala [hunk 3]
```diff
@@ -23,18 +23,18 @@ import javax.inject.Inject
...
 class MucrOptionsController @Inject()(
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  mcc: MessagesControllerComponents,
-  movementRepository: MovementRepository,
-  mucrOptionsPage: mucr_options
+                                       authenticate: AuthenticatedAction,
+                                       getJourney: JourneyRefiner,
+                                       mcc: MessagesControllerComponents,
+                                       movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/consolidations/MucrOptionsController.scala]: And here the same. And probably in more controllers :)

File: app/controllers/consolidations/ShutMucrController.scala [hunk 4]
```diff
@@ -23,29 +23,29 @@ import javax.inject.{Inject, Singleton}
...
 class ShutMucrController @Inject()(
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  mcc: MessagesControllerComponents,
-  movementRepository: MovementRepository,
-  shutMucrPage: shut_mucr
+                                    authenticate: AuthenticatedAction,
+                                    getJourney: JourneyRefiner,
+                                    mcc: MessagesControllerComponents,
+                                    movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/consolidations/ShutMucrController.scala]: Here the same

File: app/controllers/movements/GoodsDepartedController.scala [hunk 5]
```diff
@@ -24,19 +24,19 @@ import models.cache.{Cache, DepartureAnswers, JourneyType}
...
 @Singleton
 class GoodsDepartedController @Inject()(
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  movementRepository: MovementRepository,
-  mcc: MessagesControllerComponents,
-  goodsDepartedPage: goods_departed
+                                         authenticate: AuthenticatedAction,
+                                         getJourney: JourneyRefiner,
+                                         movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/movements/GoodsDepartedController.scala]: and here

File: app/controllers/movements/MovementDetailsController.scala [hunk 6]
```diff
@@ -25,21 +25,21 @@ import play.api.data.Form
...
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  movementRepository: MovementRepository,
-  mcc: MessagesControllerComponents,
-  details: MovementDetails,
-  arrivalDetailsPage: arrival_details,
-  departureDetailsPage: departure_details
+                                           authenticate: AuthenticatedAction,
+                                           getJourney: JourneyRefiner,
+                                           movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/movements/MovementDetailsController.scala]: and here

File: app/controllers/movements/MovementSummaryController.scala [hunk 7]
```diff
@@ -31,13 +31,13 @@ import views.html.summary.{arrival_summary_page, departure_summary_page}
...
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  movementRepository: MovementRepository,
-  submissionService: SubmissionService,
-  mcc: MessagesControllerComponents,
-  arrivalSummaryPage: arrival_summary_page,
-  departureSummaryPage: departure_summary_page
+                                           authenticate: AuthenticatedAction,
+                                           getJourney: JourneyRefiner,
+                                           movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/movements/MovementSummaryController.scala]: and here

File: app/controllers/movements/TransportController.scala [hunk 8]
```diff
@@ -35,7 +35,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class TransportController @Inject()(
   authenticate: AuthenticatedAction,
   getJourney: JourneyRefiner,
-  movementRepository: MovementRepository,
+  movementRepository: CacheRepository,
```
Comment (PatrykRudnicki) [inline app/controllers/movements/TransportController.scala]: and here

File: app/controllers/consolidations/DisassociateUCRSummaryController.scala (hunk 2 above)
Comment (ed0906) [reply app/controllers/consolidations/DisassociateUCRSummaryController.scala]: Yeh think I've missed a few of these

File: app/controllers/movements/TransportController.scala (hunk 8 above)
Comment (ed0906) [reply app/controllers/movements/TransportController.scala]: Fixed

File: app/controllers/movements/MovementSummaryController.scala (hunk 7 above)
Comment (ed0906) [reply app/controllers/movements/MovementSummaryController.scala]: Fixed

File: app/controllers/movements/MovementDetailsController.scala (hunk 6 above)
Comment (ed0906) [reply app/controllers/movements/MovementDetailsController.scala]: Fixed

File: app/controllers/movements/GoodsDepartedController.scala (hunk 5 above)
Comment (ed0906) [reply app/controllers/movements/GoodsDepartedController.scala]: Fixed

File: app/controllers/consolidations/ShutMucrController.scala (hunk 4 above)
Comment (ed0906) [reply app/controllers/consolidations/ShutMucrController.scala]: Fixed

File: app/controllers/consolidations/MucrOptionsController.scala (hunk 3 above)
Comment (ed0906) [reply app/controllers/consolidations/MucrOptionsController.scala]: Fixed

File: app/controllers/consolidations/AssociateUCRController.scala (hunk 1 above)
Comment (ed0906) [reply app/controllers/consolidations/AssociateUCRController.scala]: Fixed

**Pull request merged**